lms/djangoapps/grades/signals/handlers.py

```diff
@@ -158,14 +159,13 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
     Handles the PROBLEM_WEIGHTED_SCORE_CHANGED signal by
     enqueueing a subsection update operation to occur asynchronously.
     """
-    result = recalculate_subsection_grade.apply_async(
+    result = _recalculate_subsection_grade.apply_async(
```

Comment:
It's part of the temporary shim to clean up the `recalculate_subsection_grade` handler. The new, correct method name is temporarily being labelled `_name` for this release, so that pre-existing tasks queued against `name` can still be run. However, new code can call the correct new method, and does so here.

The intent is to remove the shim and rename the new method to be just `recalculate_subsection_grade` (with a single `**kwargs` in the method signature) over the next 2 releases.